viewer/packages/rendering/src/glsl/math/colorSpaceConversion.glsl

```diff
@@ -0,0 +1,7 @@
+vec4 LinearTosRGB( in vec4 value ) {
```

Comment:
Is this code from StackOverflow or somewhere? Would be nice to have a URL to the source

Reply:
will add a reference or explanation